src/autofill-utils.js

```diff
@@ -55,8 +55,9 @@ const autofillEnabled = (globalConfig) => {
     if (!userPreferences) return false;
 
     // Check config on Apple platforms
-    // @ts-ignore - TODO: C-S-S must be migrated to use the config from privacy-configuration
-    const processedConfig = processConfig(contentScope, userUnprotectedDomains, userPreferences);
+    const contentScopeAsAny = /** @type {any} */ (contentScope);
+    const cssUserPreferences = /** @type {any} */ (userPreferences);
+    const processedConfig = processConfig(contentScopeAsAny, userUnprotectedDomains, cssUserPreferences);
     return isAutofillEnabledFromProcessedConfig(processedConfig);
```

Comment:
here we interface with an external lib (css) where the types are incorrect. so, the temporary solution is to set the vars as `any` instead of ts-ignore